addons/website_event_sale/tests/test_website_event_sale.py

```diff
@@ -5,16 +5,16 @@
 
 class TestWebsiteEventSale(HttpCaseWithUserPortal, TestWebsiteEventSaleCommon):
 
-    def test_website_event_sale(self):
-        """ Test saleorder is created for tickets having price only """
-        self.authenticate('portal', 'portal')
+    def test_website_event_sale_free_tickets(self):
+        """ Test saleorder is not created for tickets free tickets """
+        self.authenticate(None, None)
```

Comment:
Does it have any impact ? Side question: if any, can't we test as portal and public to check notably for acls ?

Reply:
It should not, the process doesn't depend on the user.
I just removed the portal session (on a suggestion of @reth-odoo) to match with a more realistic case.

Reply:
Ok :)